### ISSUE TITLE
.NET Core returned  a Adar A in a non-leap year

### ISSUE THREAD

**Issue by leecow** [dotnet/corefx#27564]
@MosheL commented on [Wed Feb 28 2018](https://github.com/dotnet/core/issues/1314)

# Issue Title

Adar (אדר) month name on the .net core is displayed as a Adar A (אדר א')

# General

Example code:

		static System.Globalization.HebrewCalendar hc = new System.Globalization.HebrewCalendar();
		static System.Globalization.CultureInfo jewishCulture = 
                System.Globalization.CultureInfo.CreateSpecificCulture("he-IL");

		static public string toHebrewString(DateTime dDate)
		{
			var s= dDate.ToString("dd בMMMM y", jewishCulture);
			return s;
		}
...

           toHebrewString(new DateTime(2018,2,28) // י"ג באדר א' תשע"ח

28/2/2018 is  in Adar( י"ג באדר), not in  Adar A (י"ג באדר א'), as  תשע"ח  is not a leap year.

On Windows  10 machine the correct month (אדר) is displayed.

**Comment by danmosemsft** [comment]
On Unix we rely on http://cldr.unicode.org/ data. Is it correct there?

**Comment by danmosemsft** [comment]
That was directed to @MosheL

**Comment by tarekgh** [comment]
This looks a bug we need to look at.

**Comment by MosheL** [comment]
I am a new on .NET Core, so I do not know how it  managed inside, but I am looking now on their website and see  this:

http://cldr.unicode.org/development/development-process/design-proposals/hebrew-months

They are displaying Adar II  on leap year as Adar, this is wrong. On a leap year we need ti display a Adar I And Adar II.

**Comment by tarekgh** [comment]
the article is saying 

```
Here's what our Hebrew contacts are telling us about month names and numbers in the Hebrew calendar:

From an end-user's point of view, the numbering of Hebrew months is always consecutive.  Even though the numbers are seldom ( if ever ) used in practice.  That is to say, in a non-leap year:

Shevat = month 5, Adar = month 6, Nisan = Month 7

while in a leap year:

Shevat = month 5, Adar I = month 6, Adar II = month 7,  and Nisan = month 8.
```

which is matching what you are saying. I mean what is written is correct. the rest of the article is just trying to suggest how the data can be stored and then the data consumer can handle the list for leap and non-leap years.

**Comment by tarekgh** [comment]
The fix should be in next 2.1 preview release (i.e. in preview 2). Also, can try the fix on the master branch from now.
